plane/source/docs/quadplane-flying.rst

```diff
@@ -7,47 +7,58 @@ While flying a QuadPlane can actually be easier than flying a
 conventional fixed wing aircraft there are some things you need to
 understand. Please read the following sections carefully.
 
-Transition
-==========
+Transitions
+===========
 
 QuadPlane transition is where the aircraft is changing between flying
 primarily as a VTOL (copter-like) aircraft and flying as a
 conventional fixed wing aircraft. Transition happens in both
 directions, and can either be commanded by the pilot or happen
-automatically based on airspeed and flight mode.
+automatically based on airspeed/attitude/altitude(:ref:`assisted_fixed_wing_flight` or flight mode changes during a mission.
```

Comment:
you missing a `)` in there somewhere, or you have a extra `(`

Reply:
thanks..thought I had fixed that